flood_adapt/api/output.py

```diff
@@ -1,6 +1,8 @@
+from pathlib import Path
 from typing import Any
 
 import pandas as pd
```

Comment:
Please run "ruff check --fix ." in the project folder.
This will show you all the linting errors and will try to fix it for you.
 and "black ." to do automatic formatting.

Suggestion: maybe we should try implementing pre-commit hooks, such that these standard actions are triggered automatically when committing.